docs/nodes/nodes-library/nodes/Google/Google Tasks/README.md

```diff
@@ -0,0 +1,40 @@
+---
+permalink: /nodes/n8n-nodes-base.googleTasks
+---
+
+# Google Tasks
+
+[Google Tasks](https://www.google.com/tasks/) is a task management service developed by Google.It keeps track of your daily tasks, organize multiple lists, and track important deadlines.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Google/README.md).
```

Comment:
Please make sure that the link here works after you change the folder structure. I would recommend running the project locally to check that.